app.js

```diff
@@ -115,6 +115,11 @@ applyProxy(app, {
   ]
 });
 
+applyProxy(app, {
+  source: '/refdata/**'
+  target: config.get('proxy.refdata')
```

Comment:
has Reference Data anything to do with CCD? 
Why ExUI needs to access that backend through the CCD Gateway?

Reply:
It is tactical solution until ref data has it own API gateway.